packages/user/Fractals/service/src/tables/fractal.rs

```diff
@@ -88,12 +99,28 @@ impl Fractal {
         new_instance
     }
 
+    pub fn init_token(&mut self) {
+        let total_supply = Tokens::call().getToken(self.token_id).max_issued_supply;
+        let quarter_supply: Quantity = (total_supply.value / 4).into();
```

Comment:
So this is leaving "teams" (I assume) out as a possibility...

I do remember discussing this with you and landing on 4 distributions, but I'm now thinking maybe we should leave this at 5 distributions, and if we later decide we don't want the team distribution, then we can divide it equally and split it into each of the other token streams. Whereas if we make it 4 for now, then it's basically permanent.